src/utils/user_id.js

```diff
@@ -0,0 +1,17 @@
+import jwt_decode from 'jwt-decode';
+
+
+export const getUser_uid = () =>
```

Comment:
Nitpick: You have mixed camelCase with snake_case here. Let's choose one and stick to it.